bedrock/firefox/templates/firefox/index.html

```diff
@@ -1,157 +0,0 @@
-{#
- This Source Code Form is subject to the terms of the Mozilla Public
- License, v. 2.0. If a copy of the MPL was not distributed with this
- file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#}
-
-{% extends "firefox/base/base-protocol.html" %}
-{% from "macros.html" import google_play_button, apple_app_store_button, fxa_email_form with context %}
-
-{# Issue 13019: Avoid duplicate content for English pages. #}
-{%- block page_title_full -%}
-  {%- if LANG == 'en-US' -%}
-    Get Firefox browser — Mozilla (US)
-  {%- elif LANG == 'en-GB' -%}
-    Download Firefox Browser — Mozilla (UK)
-  {%- else -%}
-    {{ ftl('firefox-browsers-page-title') }} - Mozilla
-  {%- endif -%}
-{%- endblock -%}
-
-{% block page_desc %}{{ ftl('firefox-browsers-page-desc') }}{% endblock %}
-
-{% set show_firefox_app_store_banner = switch('firefox-app-store-banner') %}
-
-{% block page_css %}
-  {{ css_bundle('protocol-emphasis-box')}}
-  {{ css_bundle('firefox-browsers-products') }}
-
-  {% if show_firefox_app_store_banner %}
-    {{ css_bundle('firefox-app-store-banner') }}
-  {% endif %}
-{% endblock %}
-
-{% block page_banner %}
-  {% if show_firefox_app_store_banner %}
-    {% include 'includes/banners/mobile/firefox-app-store.html' %}
-  {% endif %}
-{% endblock %}
-
-{% set android_url = play_store_url('firefox', 'firefox-home') %}
-{% set ios_url = app_store_url('firefox', 'firefox-home') %}
-{% set referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=firefox-home' %}
-{% set _entrypoint = 'mozilla.org-firefox-home' %}
-
-{% block sub_navigation %}
-  {% include 'firefox/includes/sub-nav-firefox.html' %}
-{% endblock %}
-
-{% block content %}
-<main>
-  <header class="c-main-header mzp-l-content mzp-t-content-md">
-    <div class="mzp-c-logo mzp-t-logo-lg mzp-t-product-firefox"></div>
-    {% if ftl_has_messages('firefox-browsers-main-heading') %}
-      <h1>{{ ftl('firefox-browsers-main-heading') }}</h1>
-      <h2 class="mzp-has-zap-11">
-        {{ ftl('firefox-browsers-get-the-browsers-strong-v2') }}
-      </h2>
-    {% else %}
-      <h1 class="mzp-has-zap-11">
-        {{ ftl('firefox-browsers-get-the-browsers-strong-v2') }}
-      </h1>
-    {% endif %}
-  </header>
-
-  <div class="mzp-l-content c-landing-grid">
-    <div class="c-landing-grid-item">
-      {{ resp_img(
-        url='img/firefox/browsers/desktop-v2.jpg',
-        srcset={
-          'img/firefox/browsers/desktop-high-res-v2.jpg': '2x'
```

Comment:
These three tiles (six images) are most likely the last occurrence.

Unless some of them is kept to repurpose for, say #16011, they all can go along with the page.